lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -189,13 +200,31 @@ def self.available_options
 
           FastlaneCore::ConfigItem.new(key: :app_name,
                                   env_name: "APPCENTER_APP_NAME",
-                               description: "App name. If there is no app with such name, you will be prompted to create one",
+                               description: "App slug. If there is no app with such slug, you will be prompted to create one",
```

Comment:
This is not the slug. `<owner_name>/<app_name>` is the slug

Reply:
Oh, thank you for note